{{cookiecutter.project_slug}}/pyproject.toml

```
@@ -131,4 +136,9 @@ legacy_tox_ini = """
     ) %}
         py3{{python_version}}
     {%- endfor %}
+
+    [testenv:docs]
+    commands =
+        mkdocs build -s
+    extras = docs
```

Unnecessary, but for consistency. Would also advocate for this section being moved to line 131
```suggestion
    extras =
        docs
```